ci-operator/config/openshift-psap/ci-artifacts/openshift-psap-ci-artifacts-release-next.yaml

```diff
@@ -20,17 +20,17 @@ resources:
       cpu: 2000m
       memory: 2Gi
 tag_specification:
-  name: "4.6"
+  name: "4.7"
   namespace: ocp
 tests:
 - as: gpu-operator-e2e
-  postsubmit: true
+  cron: 0 */23 * * *
   steps:
     cluster_profile: aws
     test:
     - as: nightly
       cli: latest
-      commands: run gpu-ci
+      commands: run gpu-commit-ci
```

Comment:
I don't see `gpu-commit-ci` as an option at https://github.com/openshift-psap/ci-artifacts/blob/release-next/build/root/usr/local/bin/run#L48 ?

Reply:
not yet there, but it's only its way:

https://github.com/openshift-psap/ci-artifacts/pull/26/files#diff-5ac85e504d9fba79fc112fb78c18befbcfad19a10bffe0a5ebc963ba79154778R49

only thing is that I need to update this PR ^^^ to set the default repository/branch to test, so that it's not hardcoded in this `release` repo